cnd/src/swap_protocols/halight.rs

```diff
@@ -34,6 +36,16 @@ pub struct CreatedSwap {
     pub cltv_expiry: u32,
 }
 
+/// Halight specific data for an in progress swap.
+#[derive(Debug, Clone, Copy)]
+pub struct InProgressSwap {
+    pub ledger: lightning::Regtest,
+    pub asset: asset::Bitcoin,
+    pub refund_identity: identity::Lightning,
+    pub redeem_identity: identity::Lightning,
+    pub expiry: Timestamp,
```

Comment:
Given the [comment that there should be 2 expiries (`cltv_exiry` and `expiry`)](https://github.com/comit-network/comit-rs/pull/2595/files#r422750934) I suppose we should add the `cltv_expiry` here.
If we only supply one expiry on the API level I would opt for setting this to the same expiry internally.